annotate.html

```diff
@@ -0,0 +1,411 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN"
+   "http://www.w3.org/TR/html4/loose.dtd">
+<html>
+  <head>
+<META NAME="viewport" CONTENT="width=device-width, initial-scale=1.0">
+    <title>Metamath $j annotations</title>
+  </head>
+  <body>
+<TABLE BORDER=0 WIDTH="100%"><TR>
+<!-- ...TD style="line-height:100%"... -->
+
+<TD NOWRAP ALIGN=LEFT VALIGN=MIDDLE WIDTH="25%"> <FONT SIZE=-2
+FACE=arial><A HREF="../index.html"><IMG SRC="mm.gif" BORDER=0
+ALT="Home" HEIGHT=32 WIDTH=32 ALIGN=LEFT><IMG SRC="spacer.gif" BORDER=0
+ALT="" HEIGHT=32 WIDTH=1 ALIGN=LEFT>Metamath<BR>Home</A></FONT></TD>
+
+<TD NOWRAP ALIGN=CENTER VALIGN=MIDDLE><FONT SIZE="+3"
+COLOR="#006633"><B>Metamath $j annotations</B></FONT></TD>
+
+<TD WIDTH="25%"> &nbsp; </TD>
+
+</TR></TABLE>
+
+<HR NOSHADE SIZE=1>
+<p>As noted in section 4.4.3, "Additional Information Comment ($j)",
+of the metamath specification, we have a way to include information in
+metamath proof files which are not needed for verifying, but which are
+useful for other tools like proof assistants, definition checkers,
+conversion to and from other proof formats, and the like. Although the
+metamath specification has some examples, it seems better to document
+the full list here because they aren't part of the specification in
+the sense of being needed for verification.</p>
+
+<p>Many of these are experimental and may be used by one or a few
+tools.  Or in many cases, are intended for future use by tools but
+the tools haven't been written yet.  Be prepared to look at the
+documentation (or source code) for a particular tool before you make
+too many assumptions about what these comments do, or whether they do
```

Comment:
```suggestion
too many assumptions about what these commands do, or whether they do
```
We should try to be consistent with naming: these are "$j commands" or just "commands" rather than comments.

Reply:
Note that the metamath book calls them "$j comments" and things inside the $j are referred to as "commands", for instance, it writes:

>Within an additional information comment is a sequence of one or more commands of the form `command arg arg ... ;`

Reply:
I think "$j comment" is the descriptor for a comment unit like `$( $j ... $)`, while $j command is the term for a command like `syntax "foo";` that can appear in a $j comment.

Reply:
Sounds right. In the words of the metamath book, "Within an additional information comment is a sequence of one or more commands of the form [goes on to give details]".

I took a pass to speak more consistently of `$j commands`, to mostly drop the word "comments" and to completely drop the word "annotations" (which I sort of like but not enough to propose displacing existing terminology).